### ISSUE TITLE
Regarding attribute usage

### ISSUE THREAD

**Issue by bargisou** [ManiAm/VENTOS_Public#40]
Is it possible to define "member_index" of vehicle_flow by using "begin", "type", "departLane" attributes?

**Comment by bargisou** [comment]
Sir is it possible to define an individual vehicle in add node by adding them separately on different lanes and possition with different speed and using member index to define vehicle flow of various class of vehicles?

**Comment by bargisou** [comment]
Is it not possible to insert only one  vehicle as individual in addnode and inside each addnode different vehicles placed at different positions?

**Comment by ManiAm** [comment]
>> Is it possible to define "member_index" of vehicle_flow by using "begin", "type", "departLane" attributes?

member index is only for platoon and is not supported in vehicle_flow.

>> Is it not possible to insert only one vehicle as individual in addnode and inside each addnode different vehicles placed at different positions?

Why not using multiple vehicle attribute one after the other each with different departPos?

**Comment by bargisou** [comment]
Yes...I have tried using that...But it is not possible to insert a type=TypeCACC for individual vehicle.
-- Is it that a single CACC vehicle cannot be inserted?  
-- Could you tell me if it is possible to apply lane changing for an individual vehicle in addnode?
-- As in, if I use different vehicle attributes and define the type=TypeManual can I make that vehicle to change that lane ?

**Comment by bargisou** [comment]
My actual requirement is to make the individual vehicle change lane.
For example: If a vehicle is inserted on 
departLane=1 then make it move on departLane =2 after 30sec may be.
Can this be possible in addnode?
Or is there any possible way to do so ?

**Comment by bargisou** [comment]
Are parameters like arrivalLane, arrivalpos, arrivalSpeed available for usage in VENTOS?

**Comment by ManiAm** [comment]
>> Are parameters like arrivalLane, arrivalpos, arrivalSpeed available for usage in VENTOS?

No. You need to use the traffic demand file of SUMO directly and create vehicles directly there.

>> My actual requirement is to make the individual vehicle change lane.

You need to code that behavior in C++ through the TraCI interface.
